plugins/modules/web_infrastructure/apache2_module.py

```diff
@@ -49,6 +49,8 @@
      type: bool
      default: False
 requirements: ["a2enmod","a2dismod"]
+notes:
+  - Currently only systems based on Debian and SuSE support the C(a2enmod) and C(a2dismod) commands.
```

Comment:
If this is just a remark of how the world looks like in CI, this should be a comment in the tests, and not a note for the module. Or do you have a reference for this that this is really only the case for Debian and SuSE?

Reply:
The module relies on a2enmod/a2dismod and those commands do not exist in the httpd installation on redhat-flavored systems. Hence, the module **only** works in Debian- and SuSe-based ones, AFAICT. Haven't tested Alpine nor niche distros.

Reply:
Well, since you skipped a lot of other distributions (even if they are only niche), you should formulate this differently. Something like "This does not work on RedHat-based distributions. It does work on Debian- and SuSE-based distributions. Whether it works on others depend on whether the `a2enmod` and `a2dismod` tools are available or not."

Reply:
```
  - This does not work on RedHat-based distributions. It does work on Debian- and SuSE-based distributions. Whether it works on others depend on whether the `a2enmod` and `a2dismod` tools are available or not.
```

Reply:
You need to use C(...) instead of the single backticks though :)

Reply:
I was out and about, tried to get it done in the phone, but no luck :/

Reply:
```suggestion
  - This does not work on RedHat-based distributions. It does work on Debian- and SuSE-based distributions. Whether it works on others depend on whether the C(a2enmod) and C(a2dismod) tools are available or not.
```